acme/acme/crypto_util.py

```diff
@@ -7,8 +7,7 @@
 import socket
 
 import josepy as jose
-from OpenSSL import crypto
-from OpenSSL import SSL  # type: ignore # https://github.com/python/typeshed/issues/2052
+from OpenSSL import crypto, SSL  # type: ignore # https://github.com/python/typeshed/issues/2052
```

Comment:
I'll check if this change is necessary. I think the linter was complaining about this.